pkg/provider/cloud/kubevirt/provider.go

```diff
@@ -34,16 +34,23 @@ import (
 const (
 	// FinalizerNamespace will ensure the deletion of the dedicated namespace.
 	FinalizerNamespace = "kubermatic.k8c.io/cleanup-kubevirt-namespace"
+	// KubevirtDiskCacheNamespace represents namespace for kubevirt-disks.
```

Comment:
Maybe we should add a Finalizer for:
- dataVolumeRoleBinding that is created in the `kubevirt-disks` namespace
to ensure that it's delete before delete the cluster.